app/repositories.php

```diff
@@ -27,23 +27,34 @@
 return static function (ContainerBuilder $containerBuilder) {
     $containerBuilder->addDefinitions([
         CampaignFundingRepository::class => static function (ContainerInterface $c): CampaignFundingRepository {
-            return $c->get(EntityManagerInterface::class)->getRepository(CampaignFunding::class);
+            $repository = $c->get(EntityManagerInterface::class)->getRepository(CampaignFunding::class);
```

Comment:
but here all Psalm knows is that repository is an instance of `EntityManagerInterface<CampaignFunding>`, so if we need it to be more specific than that we have to assert it.